pkg/services/ngalert/store/database_mig.go

```diff
@@ -107,3 +107,27 @@ func alertInstanceMigration(mg *migrator.Migrator) {
 	mg.AddMigration("add index in alert_instance table on def_org_id, def_uid and current_state columns", migrator.NewAddIndexMigration(alertInstance, alertInstance.Indices[0]))
 	mg.AddMigration("add index in alert_instance table on def_org_id, current_state columns", migrator.NewAddIndexMigration(alertInstance, alertInstance.Indices[1]))
 }
+
+func SilenceMigration(mg *migrator.Migrator) {
```

Comment:
Are we going to keep this table around (also the commands to use the table below)? Given we're not going down this route (for now) I would prefer us to end up with this code to avoid confusion - could we create an issue to follow up on its removal?

Reply:
good catch! it was accidentally merged and we don't want to keep this migration.
Since the migration is behind a feature flag, I guess it's fine to just remove it, right?
The cons doing so it's that if somebody has enabled the feature and the table is created, then they have to delete it manually (the alternative would be to keep the migration and delete the table at the end but personally I prefer the other option).

Reply:
I'm in favour of just removing it. This was merged a couple of hours ago, and the table is not internally used - for features behind a flag I guess we don't make any guarantees right @torkelo?

Reply:
I have opened #32595; let's continue the discussion there